public/v4/apps/openspeedtest.yml

```diff
@@ -2,23 +2,23 @@ captainVersion: 4
 
 services:
     '$$cap_appname':
-        image: index.docker.io/openspeedtest/latest:$$cap_openspeedtest_version
+        image: index.docker.io/test/latest:$$cap_test_version
         caproverExtra:
             containerHttpPort: '3001'
         restart: always
 
 caproverOneClickApp:
     variables:
-        - id: '$$cap_openspeedtest_version'
-          label: OpenSpeedTest Version
+        - id: '$$cap_test_version'
+          label: Test Version
           defaultValue: 'speedtest'
-          description: Check out their Docker page for the valid tags https://hub.docker.com/r/openspeedtest/latest/tags
+          description: Check out their Docker page for the valid tags https://hub.docker.com/r/test/latest/tags
     instructions:
         start: |-
-            OpenSpeedTest is a self-hosted speedtest. You can read more about this on https://openspeedtest.com/selfhosted-speedtest.
+            Test is a self-hosted speedtest. You can read more about this on https://test.com/selfhosted-speedtest.
```

Comment:
same here... **Test** is a self-hosted speedtest. You can read more about this on https://**test.com**/selfhosted-speedtest.

Reply:
super weird. the only thing i did was run the formatter. i fixed it with the next push